### DISCUSSION TITLE
Change VCS(git) Repository branch and Push branch

### DISCUSSION THREAD

**Discussion by wirthandras** [WeblateOrg/weblate#7525] [Q&A]
Hi,

I see I can perform repository pull, push actions via Rest API as the following way:

https://docs.weblate.org/en/latest/api.html#post--api-projects-(string-project)-repository-

Is there any way via Rest API to change the Repository branch or Push branch value?

**Comment by nijel** [comment]
You can do that in component settings: https://docs.weblate.org/en/latest/api.html#patch--api-components-(string-project)-(string-component)-
